docs/canonicalk8s/reuse/substitutions.yaml

```diff
@@ -1,6 +1,6 @@
 product: 'Canonical Kubernetes'
-version: '1.31'
-channel: '1.31/candidate'
+version: '1.32'
+channel: '1.32/stable'
```

Comment:
Why do we define those channels here but still need to change the channels in a bunch of places?

Reply:
These {{channels}}/{{version}} cannot be included in code snippets so that is why we have the install file instead. I'm not sure if they will be used in the future.